stdlib/public/core/BridgeObjectiveC.swift

```diff
@@ -85,6 +85,18 @@ public protocol _ObjectiveCBridgeable {
 
 #if _runtime(_ObjC)
 
+@available(*, unavailable)
```

Comment:
Hah, so IIUC, this makes these only accessible through C. Very nice!

Reply:
I do not trust this; the compiler might say "okay, to save on code size I am going to remove it". (Consider all of the APIs that are unavailable on a particular platform.)

Reply:
Hmmmm :( dangit, this so neatly solved the "make the ABI checker happy" and "make sure nobody abuses this" problems.

Reply:
The S4TF folks have a nice solution for this kind of thing - they use “func _(...)” for their differentiation thunks. You can’t directly call them from Swift (since they don’t have a name), but their attributes expose them to the compiler in other ways.

I don’t know if unnamed functions are allowed on master, though, or if we’d need a swift-evo proposal to add them.

Reply:
That wouldn't work with module interfaces, which still have to parse like normal source files (and the S4TF folks are going to have to come up with something else for that).

Reply:
Given that it's already `internal` just putting a deprecation on it is probably good enough.